### PR TITLE
Convert packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/Scrubber.test.js to RTL

### LINKED ISSUE #50402: Convert `packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/Scrubber.test.js` to RTL

### PR DESCRIPTION
**Pull request by gitstart**
**What this PR does / why we need it**:
Convert Scrubber.test.js to RTL

**Which issue(s) this PR fixes**:

Fixes #50402

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by gitstart** [comment]
@zoltanbedi PR is ready for review, please review 🙏

**Comment by zoltanbedi** [comment]
Hey @gitstart, thanks for creating this PR. I'm currently on parental leave so I'm going to ask @joey-grafana to review this for me.

File: packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/Scrubber.tsx [hunk 1]
```diff
@@ -85,30 +85,41 @@ export default function Scrubber({ isDragging, onMouseDown, onMouseEnter, onMous
...
-          width="9"
-          height="20"
-        />
-        <rect
-          x={xPercent}
-          className={styles.ScrubberHandle}
-          style={{ transform: `translate(-1.5px)` }}
-          width="3"
-          height="20"
+    <svg>
```
Comment (aocenas) [inline packages/jaeger-ui-components/src/TracePageHeader/SpanGraph/Scrubber.tsx:88]: Why this added `<svg>` element? This component is already embedded in another `<svg>` in the parent so it does not seem like it should be here as it creates `<svg><svg></svg></svg>`.

**Comment by gitstart** [comment]
Hello @aocenas thank you for the review. The code has been updated, please take a look

**Review by aocenas — APPROVED** [review]

**Comment by aocenas** [comment]
@gitstart not sure if I am talking to a bot or not but thanks for the contribution :)

**Comment by myerekapan** [comment]
> 

Hey, @aocenas you’re welcome! Mert here from GitStart where we’re experimenting with contributing open source as part of our career accelerator. At GitStart, we accelerate development teams’ capacity and our engineers’ careers at the same time. There are several people helping you out via this bot - more on this [here](https://www.gitstart.com/posts/we-raised-5m-to-enable-anyone-to-grow-on-the-worlds-best-codebases#:~:text=Engineering%20a%20solution) if you’re curious.

**Pull request merged**